### DISCUSSION TITLE
这个支持多台机器分布式部署吗

### DISCUSSION THREAD

**Discussion by bytemode** [huahua132/skynet_fly#8] [Q&A]
如果想多台机器部署做分布式 这个支持吗

**Comment by huahua132** [comment]
支持的，有cluster远程调用

**Comment by bytemode** [comment]
再次请教下：
有分布式部署的实例吗 例如cheinese_chess这个例子，我想在另外一台机器上增加大厅服和与游戏服该怎么做呢？ 
或者说就是在另外的机器部署一个新的节点里面包含整组服务？

**Comment by huahua132** [comment]
可以看看examples/cluster相关的用例。cluster只是提供了多实例间通信的支持。如果多实例部署，应该增加一个单例的网关服，负责负载均衡，token下发。不过这种偏业务架构的还是要自己去考量。
